tests/baselines/reference/goToTypeDefinition_arrayType.baseline.jsonc

```diff
@@ -198,7 +198,7 @@
 // }|>
 // 
 // interface ArrayConstructor {
-//     new(arrayLength?: number): any[];
+//     new (arrayLength?: number): any[];
```

Comment:
Despite my best efforts, there's still a baseline which copies info out of `lib.d.ts` and is format sensitive. Really annoying.